fuse/src/main/java/com/github/kittinunf/fuse/core/cache/DiskCache.kt

```diff
@@ -37,7 +38,7 @@ internal class DiskCache private constructor(private val cache: DiskLruCache) :
     }
 
     override fun allKeys(): Set<String> = allSafeKeys()
-        .map { get(it, KeyType.Key.ordinal)!!.toString(Charset.defaultCharset()) }
+        .map { getEntry(it)!!.key }
```

Comment:
[q] maybe empty set when null? or do you want to keep it explicitly breaking?

Reply:
🤔 Let me think about what I can do for this